rspec-mode.el

```diff
@@ -146,6 +146,11 @@ Not used when running specs using Zeus or Spring."
   :type 'boolean
   :group 'rspec-mode)
 
+(defcustom rspec-zeus-sock-file (concat (rspec-project-root) ".zeus.sock")
```

Comment:
Note that this definition is broken by itself, because the default value is evaluated at the time `rspec-mode.el` is loaded. What would `(rspec-project-root)` return then?